### DISCUSSION TITLE
Option to use Win 11 taskbar + never combine taskbar icons?

### DISCUSSION THREAD

**Discussion by guenniguenzelsen** [valinet/ExplorerPatcher#850] [Q&A] [answered]
I'm looking for an option to use the standard Win 11 taskbar, but just have the feature to never combine taskbar icons (like in Win 10 taskbar).
Is this possible actually? Or is there a chance to integrate?

Thanks!
Guenni

**Comment by valinet** [comment, marked as answer]
W11 taskbar is much more locked down than W10 one. Some people are looking into it more in-depth, I think, like the developer of 7+TT. Maybe he comes up with something, I personally am not working on anything regarding it at the moment.

Best is to ask Microsoft since it’s their implementation after all.

Thanks

**Comment by guenniguenzelsen** [comment]
Thanks for your clear response and linking me to the alternative developers
